src/harvest-rpde.js

```diff
@@ -200,6 +203,7 @@ async function harvestRPDE({
         return;
       }
       if (error.response?.status === 404) {
```

Comment:
Perhaps for consistency with the spec?

```suggestion
      if (error.response?.status === 404 || error.response?.status === 410) {
```